chainbase/src/main/java/org/tron/core/net/message/MessageTypes.java

```diff
@@ -33,6 +33,10 @@
 
   TRX_INVENTORY(0x13),
 
+  PBFT_BLOCK_MSG(0x14),
```

Comment:
You'd better start a new message type scope. Start your `PBFT_` message type from 0x40.